Apps/Playground/Android/app/CMakeLists.txt

```diff
@@ -29,22 +29,24 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17")
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/src/")
 
 set(BABYLON_NATIVE_TESTAPP_DIR "${CMAKE_CURRENT_LIST_DIR}/../..")
-add_subdirectory(${BABYLON_NATIVE_TESTAPP_DIR}/../../Library/ ${BABYLON_NATIVE_TESTAPP_DIR}/../../build/Android_${CMAKE_ANDROID_ARCH_ABI}/)
+add_subdirectory(${BABYLON_NATIVE_TESTAPP_DIR}/../../Dependencies/)
+add_subdirectory(${BABYLON_NATIVE_TESTAPP_DIR}/../../Core/)
+add_subdirectory(${BABYLON_NATIVE_TESTAPP_DIR}/../../Plugins/)
+add_subdirectory(${BABYLON_NATIVE_TESTAPP_DIR}/../../build/Android_${CMAKE_ANDROID_ARCH_ABI}/)
```

Comment:
This might not be necessary anymore.

Reply:
Actually binary dir. https://cmake.org/cmake/help/v3.0/command/add_subdirectory.html